files/faban/src/cloud/benchflow/collectors/faban.go

```diff
@@ -51,8 +51,8 @@ func signalOnKafka(minioKey string, containerID string) {
 	}
  
 func backupHandler(w http.ResponseWriter, r *http.Request) {
-    query := r.FormValue("trial_id")
-    path := "/app/faban/"+query
+    query := r.FormValue("path")
```

Comment:
Why are query? only the folder inside the `/app/faban/output/` is passed as parameter as pointed out in https://github.com/benchflow/collectors/pull/63#discussion_r64148402 and https://github.com/benchflow/collectors/issues/36. Set `/app/faban/output/` as `file` variable, and just add the folder you get from the query to the path (adding the trailing slash).